physics/radiation_aerosols.f

```diff
@@ -4320,11 +4341,13 @@ subroutine aer_property_gocart                                    &
 
 ! --- update diagnostic aod arrays
           do k = 1, NLAY
-           aerodp(i,1) = aerodp(i,1) + tauae(k,nv_aod)
-          enddo
+           !aerodp(i,1) = aerodp(i,1) + tauae(k,nv_aod)
```

Comment:
Unless coding changes are truly temporary, remove lines rather than commenting them out. The old versions will always be saved in git history. Also applies to 4348, 4474, 4523.

Reply:
I removed those lines.